### PR TITLE
fixed: https://github.com/Sublime-Instincts/BetterTwig/issues/10

### PR DESCRIPTION
**Pull request by andremacola**
Added scopes for `endif` and `endfor`

### CODE REVIEW

File: resources/syntax/Twig.sublime-syntax [hunk 1]
```diff
@@ -103,6 +105,10 @@ contexts:
     - match: \b(if)\b
       scope: keyword.control.conditional.if.twig
 
+  match_keyword_endif:
+    - match: \b(endif)\b
+      scope: keyword.control.conditional.endif.twig
```
Comment (Ultra-Instinct-05) [inline resources/syntax/Twig.sublime-syntax:110]: IMO, this should be scoped as `keyword.control.conditional.end.twig`. Please have a look at this comment :- https://github.com/sublimehq/Packages/issues/1228#issuecomment-461077685

File: resources/syntax/Twig.sublime-syntax [hunk 2]
```diff
@@ -115,6 +121,10 @@ contexts:
     - match: \b(for)\b
       scope: keyword.control.loop.for.twig
 
+  match_keyword_endfor:
+    - match: \b(endfor)\b
+      scope: keyword.control.loop.endfor.twig
```
Comment (Ultra-Instinct-05) [inline resources/syntax/Twig.sublime-syntax:126]: Same as the previous one. Please scope it as `keyword.control.loop.end.twig`

**Comment by Ultra-Instinct-05** [comment]
Also, please change the respective test case for the scope change. Just open the syntax file and <kbd>ctrl + shift + b</kbd>, then go to `Syntax Tests`. Since the scopes have changed, there should be a 1 or 2 failing tests. Just correct/update them & ensure that the tests pass.

**Comment by andremacola** [comment]
I think now is ok. I'm kind new on sublime stuff, I didn't know that the sublime had a function to test syntax

**Pull request merged**